definitions/npm/jest_v26.x.x/flow_v0.134.x-/jest_v26.x.x.js

```diff
@@ -832,6 +832,7 @@ type JestObjectType = {
    * mocked version of the module for you.
    */
   genMockFromModule(moduleName: string): any,
+  createMockFromModule(moduleName: string): any,
```

Comment:
Looks like we generally add JS doc comments which help in the editor. Do you want to add that also? 

Could also add a test to validate the type like I did for the recent change to the file like this https://github.com/mrtnzlml/flow-typed/blob/jest_v26_createMockFromModule/definitions/npm/jest_v26.x.x/flow_v0.134.x-/test_jest-v26.x.x.js#L309

Reply:
both fixed